house/forms.py

```diff
@@ -0,0 +1,8 @@
+from django import forms
+from expenses.models import Expenses
+
+
+class ExpenseForm(forms.ModelForm):
```

Comment:
I used a different kind of Form in the Login Page
I got it from what I have read here: https://docs.djangoproject.com/en/4.0/topics/forms/#building-a-form-in-django
(Building a form in Django: The form class)

Where did you read about this kind of form?